scripts/update_nolints.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
This file needs to be renamed so it is clearly labelled as part of CI. Humans shouldn't be running this locally to update the nolints!

Reply:
Done, and left a big comment.